cmd/lncli/cmd_invoice.go

```diff
@@ -57,7 +57,8 @@ var addInvoiceCommand = cli.Command{
 		cli.Int64Flag{
 			Name: "expiry",
 			Usage: "the invoice's expiry time in seconds. If not " +
-				"specified an expiry of 3600 seconds (1 hour) " +
+				"specified, an expiry of " +
```

Comment:
nit: could probably fit this string on just 2 lines by not wrapping so early. same for below

Reply:
I understood that your suggestion is following
```suggestion
				"specified, an expiry of 86400 seconds (24 hours) is implied."
```

By this change,`line is 95 characters (lll)` is detected by go-golangci-lint.  
Could you show me any suggestion?

Reply:
looking better now! was originally spread across more lines.